migration/v2.17.x-ck8s1-v2.18.x-ck8s1/migrate-to-containerd.yml

```diff
@@ -0,0 +1,106 @@
+- hosts: k8s_cluster
+  serial: 1
+  tasks:
+
+    - name: Check that ansible version is >= 2.10.0
+      assert:
+        that:
+        - ansible_version.full is version('2.10.0', '>=')
+        fail_msg: "You need to have an ansible version >= 2.10.0"
+
+    - name: hostname
+      shell: echo $HOSTNAME
+      args:
+        executable: /bin/bash
+      register: hostname
+
+    - name: drain node
+      shell: kubectl drain {{hostname.stdout}} --ignore-daemonsets=true --delete-emptydir-data=true --force=true --kubeconfig /etc/kubernetes/admin.conf
+      delegate_to: "{{groups['kube_control_plane'][0]}}"
+
+    - name: remove docker and its dependencies
+      apt:
+        pkg:
+        - docker-ce
+        - docker-ce-cli
+        - docker-ce-rootless-extras
+        - containerd.io
+        state: absent
+        force: yes
```

Comment:
According to [this guide](https://kubespray.io/#/docs/upgrades/migrate_docker2containerd) you should stop the kubelet and docker between draining the node and removing docker.

```bash
service kubelet stop
service docker stop
```

Reply:
Done!!